### ISSUE TITLE
This app no longer works

### ISSUE THREAD

**Issue by moronimontoya** [TomFreudenberg/preziplayer#12]
Looks like Prezi has changed their export script because the zip does not come with a movie.sfw file any more.

**Comment by Tzesar** [comment]
Is there any plan to solve this issue?

**Comment by AndyGaskell** [comment]
I found Wine worked ok, it's not a great solution, but passable, happy to share the details if anyone wants

**Comment by planck2007** [comment]
Hi AndyGaskell, I would like to know how you did it using Wine. Thanks

**Comment by Xqua** [comment]
Yes it works with wine but not great !
Anychance of update ?

**Comment by tk421** [comment]
I have the same problem. An update would be nice.

**Comment by tk421** [comment]
We should send friendly request to prezi to officially support Linux. If they hear us, they might be able to change his mind.

Just go to the "Need more help" link https://prezi.com/support/search/?q=linux

**Comment by mushinako** [comment]
Yeah we should. It's kinda annoying because wine sometimes does not work well as well.

**Comment by melroy89** [comment]
Please fix it, so we can use it again..

**Comment by greymatter1** [comment]
Same problem here, with the movie.sfw file. I think this software is no longer being developed.

**Comment by sslag** [comment]
You can run the downloaded prezi without use this player.
1. Decompress the zip.
2. Go to the decompressed folder.
3. Run "wine Prezi.exe"

It allows to run presententation offline. 
Obviously, you need "wine" package to be installed previously.
First time you execute "wine Prezi.exe", stay online beacause wine will need to download some extra programs to run the adobe air app.

Enjoy it.

**Comment by greymatter1** [comment]
Wow!
That works perfectly.Thanks a ton.
Rgds,

Mateus

Date: Fri, 25 Sep 2015 03:35:23 -0700

**Comment by HansAchterbahn** [comment]
I also would love to see this app working again...
